op-service/txmgr/cli.go

```diff
@@ -274,10 +287,17 @@ func NewConfig(cfg CLIConfig, l log.Logger) (Config, error) {
 		return Config{}, fmt.Errorf("could not init signer: %w", err)
 	}
 
+	// convert float GWei value into integer Wei value
+	feeLimitThreshold, _ := new(big.Float).Mul(
+		big.NewFloat(cfg.FeeLimitThresholdGwei),
+		big.NewFloat(params.GWei)).
+		Int(nil)
```

Comment:
It can't be NaN or infty because the two numbers that are multiplied are guaranteed to be proper floats. `params.Gwei` is a constat `1e9` and `FeeLimitThresholdGwei` is a float64. The float also comes from cli flags, where it is also verified as a normal number. So I don't think we need to add this verification here.

Reply:
Are you sure the CLI verifies it to be a normal number? In the urfave v2 implementation I see it call `valFloat, err := strconv.ParseFloat(val, 64)`, of which the Go docs say that the special values are allowed:
> // ParseFloat recognizes the string "NaN", and the (possibly signed) strings "Inf" and "Infinity"
> // as their respective special floating point values. It ignores case when matching.

Or does the CLI lib verify the float edge-cases somewhere separate from value parsing?

Reply:
If someone sets the threshold to `NaN` or `Inf`, they try to break the program on purpose. I find this a rather artificial situation. I bet there are many other way more realistic ways to set other config parameters in wrong ways that break the program. But I can add it if you prefer to check for this.

Reply:
If the flag parsing gives no error, and we convert to int only later, and then get a `nil` that is unused until later, then the program panics at runtime. It's just cleaner to return an error here.

Reply:
Ajlso, the function actually doesn't return a `bool` indicating successful conversion, but it returns an `big.Accuracy`. The check would need to be done directly on `cfg.FeeLimitThresholdGwei`

Reply:
added the check

Reply:
@coderabbitai `Int` doesn't return a bool indicating successful conversion, but an accuracy.